### ISSUE TITLE
[Book 2] Section 2.4 Typo, center0 should be center1 when calculating the center of the sphere when moving

### ISSUE THREAD

**Issue by graciexia8** [RayTracing/raytracing.github.io#1333]
`center0` variable should be renamed to `center1`
Before:
```c++
    point3 center(double time) const {
        // Linearly interpolate from center1 to center2 according to time, where t=0 yields
        // center1, and t=1 yields center2.
        return center0 + time*center_vec; 
    }
```
After:
```c++
    point3 center(double time) const {
        // Linearly interpolate from center1 to center2 according to time, where t=0 yields
        // center1, and t=1 yields center2.
        return center_1 + time*center_vec; // <=== var name change here
    }
```

![image](https://github.com/RayTracing/raytracing.github.io/assets/15184219/e88dc44b-baa5-42bd-a060-29207a7a488a)

**Comment by k4lizen** [comment]
This is fixed in the dev branch

**Comment by hollasch** [comment]
Verified fixed in f4ebec4063c6b0dea23d5e480d53e07debea3927
